variants/dfrobot_lorawan_esp32s3/pins_arduino.h

```diff
@@ -0,0 +1,43 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define LED_BUILTIN 21
+
+static const uint8_t TX = 43;
+static const uint8_t RX = 44;
+
+static const uint8_t SDA = 8;
+static const uint8_t SCL = 9;
+
+#define WB_IO1 1
+#define WB_IO2 1
+#define WB_SW1 34
+#define WB_A0 36 
+#define WB_LED1 -1
+#define WB_LED2 2
```

Comment:
PTAL and check if that's correct?? 
`WB_IO1` and `WB_IO2` are both 1. Also `WB_LED1` is -1.